Assets/Prefabs/LoadingLogo/Animation/LoadingLogo Animation.anim

```diff
@@ -0,0 +1,227 @@
+%YAML 1.1
```

Comment:
For the outside ring, I liked the Bézier curve animation better than the linear one